templates/project/phpunit.xml.dist.twig

```diff
@@ -30,9 +30,14 @@ It's auto-generated by sonata-project/dev-kit package.
         <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
     </listeners>
 
-{% if project.usesPanther %}
+{% if project.usesPanther or project.usesDoctrineTest %}
     <extensions>
+{% if project.usesPanther %}
         <extension class="Symfony\Component\Panther\ServerExtension" />
+{% endif %}
+{% if project.usesDoctrineTest %}
+        <extension class="DAMA\DoctrineTestBundle\PHPUnit\PHPUnitExtension" />
```

Comment:
This extension is pretty useful, I'm kinda curious why we don't need it on DoctrineORM bundle

Reply:
Yes, To me it should be added to the projects where we use functional tests. Now we are not using it because we avoid somehow colisions between tests. On media bundle wasn't needed because of that, and classification the same. On user-bundle I can't test properly without it, because some properties are unique and it is a pain to deal with them on an non isolated environment.

Reply:
Once we have a example setup, we can use it on more bundles if needed.